src/pages/cli/graphql/troubleshooting.mdx

```diff
@@ -0,0 +1,52 @@
+export const meta = {
+  title: `Troubleshooting`,
+  description: `Add authorization rules to your GraphQL schema to control access to your data.`,
+};
+
+## Deploying multiple index changes at once
+
+You can make `@index` updates on one "amplify push". Under the hood, Amplify CLI needs to locally sequence multiple individual deployments to your DynamoDB table because each Global Secondary Index (GSI), managed by `@index`, change requires time to create the new index.
+
+If your deployment fails locally when updating multiple GSIs, you'll have the ability to run:
+
+- `amplify push --iterative-rollback` to rollback the last-known-good state
+- `amplify push --force` to rollback the last-known-good state and try redeploying your changes again using.
+
+```console
+Attempting to mutate more than 1 global secondary index at the same time.
+```
+
+If you're running into the error above during `amplify push`, it is likely that you don't have this feature enabled. To enable multiple GSI updates, set the "enableIterativeGsiUpdates" feature flag to true in your `amplify/cli.json` file.
+
+## Backfill OpenSearch index from DynamoDB table
+
+When you add `@searchable` to a `@model` type with existing data, then you need to backfill the OpenSearch index. Download the following Python script to help you backfill your OpenSearch index:
```

Comment:
We should put a warning in here that the way the script works is it pulls records from Dynamo locally then invokes the streaming lambda, depending on how they run the script this could incur some unexpected network charges